var/spack/repos/jcsda-emc/packages/fv3-jedi-linearmodel/package.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Fv3JediLinearmodel(CMakePackage):
+    """Tangent linear and adjoint versions of FV3 dynamical core and GEOS physics"""
+
+    homepage = "https://github.com/JCSDA/fv3-jedi-linearmodel"
+    git = "https://github.com/JCSDA/fv3-jedi-linearmodel.git"
+
+    maintainers = ["climbfuji"]
+
+    version("1.4.0", commit="05cc1ae63252ca535f3db0fdca9a8a996329fc8f")
+    version("1.3.0", commit="9758fbd44166fc1e1d745ca9ab7e9e5e6071955f")
+    version("1.2.0", commit="d47cea97c659e8a11e9e64c23092bef06227ebde")
+    version("develop", branch="develop", no_cache=True)
+
+    variant(
+        "forecast_model",
+        default="FV3CORE",
+        description="fv3 forecast model",
+        values=("FV3CORE", "GEOS", "UFS"),
+    )
+    # Note that the options for mkl, mpi, and openmp are lazily written in fv3-jedi-linearmodel.
+    # These options just turn on a find_package call. If a package is already found, for example by
+    # one of the package dependencies, then that component is silently used even if the user
+    # toggles it off. This is a bug and should be fixed eventually.
+    variant("mkl", default=False, description="Use MKL for LAPACK implementation (if available)")
+    variant("mpi", default=True, description="Support for MPI distributed parallelism")
+    variant("openmp", default=True, description="Build with OpenMP support")
+
+    conflicts("forecast_model=GEOS", msg="FV3-JEDI-LINEARMODEL: GEOS to be implemented.")
```

Comment:
I think we are using the current fv3-jedi-linearmodel with GEOS and UFS - @shlyaeva can you comment on this?

Reply:
You're using both, but at time of writing this PR GEOS wasn't fully buildable in Spack.

Reply:
I think we can merge it as-is now, and update later as needed